src/screens/ProfileScreen/DetailScreen/ExtensionRequest.tsx

```diff
@@ -30,8 +29,7 @@ export default function ExtensionRequest() {
       .then((responseData) => {
         console.log(JSON.stringify(responseData));
         console.log(JSON.stringify(responseData), 'PostApiCAlled');
-      })
-      .done();
```

Comment:
Why did we remove .done()?

Reply:
Checkout fetch call methods, there is no done property.
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API